dockerfiles/run/guice/memory/Dockerfile

```diff
@@ -22,7 +22,10 @@ RUN apt-get update && apt-get install -y unzip
 
 # Get data we need to run James : build results and configuration
 ADD destination/james-server-memory-guice.zip /root
-RUN unzip james-server-memory-guice.zip && cp -r /root/james-server-memory-guice/* /root
+RUN unzip james-server-memory-guice.zip && \
+  cp -r /root/james-server-memory-guice/* /root && \
+  rm james-server-memory-guice.zip && \
+  rm -r james-server-memory-guice
```

Comment:
unzip -d allows to extract to an arbitrary directory

Reply:
Yes but it do not change the fact that everything is bundled in  a `james-server-memory-guice`

I tried:

```
RUN unzip james-server-memory-guice.zip -d /root && rm james-server-memory-guice.zip
```

It deflates:

```
Step 6/21 : RUN unzip james-server-memory-guice.zip -d /root && rm james-server-memory-guice.zip
 ---> Running in 4bfb050ef3e1
Archive:  james-server-memory-guice.zip
   creating: /root/james-server-memory-guice/
   creating: /root/james-server-memory-guice/extensions-jars/
   creating: /root/james-server-memory-guice/conf/
   creating: /root/james-server-memory-guice/james-server-memory-guice.lib/
  inflating: /root/james-server-memory-guice/README.adoc  
  inflating: /root/james-server-memory-guice/conf/jmx.properties  
  inflating: /root/james-server-memory-guice/conf/listeners.xml  
  inflating: /root/james-server-memory-guice/conf/logback.xml  
```

And I got the error:

```
$ docker run -ti mem
Error: Unable to access jarfile james-server-memory-guice.jar
```

Reply:
oh right sorry for the noise. It's easily doable with tar but unfortunately not with unzip.